scripts/index.js

```diff
@@ -1,12 +1,26 @@
 
 // Implement the following functions to fulfill the tests!
-function getLocationName() {}
-function getLocationCountry() {}
-function getLocationLatitude() {}
-function getLocationLongitude() {}
-function getDescription() {}
-function getWindSpeed() {}
-function getSunrise() {}
+function getLocationName(city) {
```

Comment:
One tiny nitpick- consider making your argument names more generic. For example `weather` might be a good choice because the value being passed in...is a giant glob of weather information and not just the city :)